### PR TITLE
Temporary reagents fix

### PR DESCRIPTION
**Pull request by ninjanomnom**
This should work. Probably.

Makes reagents not able to be assigned a NaN temperature making reactions impossible. I don't know the base cause yet so here's something to troubleshoot. I can't seem to reproduce it in local testing.

### CODE REVIEW

**Comment by ninjanomnom** [comment]
wait, what? I'll figure out why that's happening

**Comment by ninjanomnom** [comment]
@monster860 It compiles just fine on my end, that error is from byond randomly deciding spacedrift.dm doesn't need to be loaded anymore.

**Comment by monster860** [comment]
@ninjanomnom No it has nothing to do with that. /datum/reagents does not have an x y or z and you are trying to get one out of it for the admin notice.

**Comment by ninjanomnom** [comment]
I still don't know why it was compiling fine on my end but here's a fix for that

File: code/modules/reagents/chemistry/holder.dm [hunk 1]
```diff
@@ -495,6 +495,9 @@ var/const/INJECT = 5 //injection
 /datum/reagents/proc/add_reagent(reagent, amount, list/data=null, reagtemp = 300, no_react = 0)
 	if(!isnum(amount) || !amount)
 		return 1
+	if(!isnum(reagtemp) || !reagtemp)
+		message_admins("[src] A reagent container received an invalid temperature var <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[usr.x];Y=[usr.y];Z=[usr.z]'>(JMP)</a>.")
```
Comment (Cirrial) [inline code/modules/reagents/chemistry/holder.dm]: message_admins("[src] A reagent container received an invalid temperature var <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[usr.x];Y=[usr.y];Z=[usr.z]'>(JMP)</a>.")

should be

message_admins("[src] A reagent container received an invalid temperature var <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[my_atom.x];Y=[my_atom.y];Z=[my_atom.z]'>(JMP)</a>.")

(i am shit at github specifically)

**Pull request merged**